src/test/scala/org/apache/spark/deploy/armada/e2e/SimplePodMonitor.scala

```diff
@@ -69,33 +71,33 @@ class SimplePodMonitor(namespace: String) {
               val lines       = describeResult.stdout.split("\n")
               val eventsIndex = lines.indexWhere(_.contains("Events:"))
               if (eventsIndex >= 0) {
-                println(s"[MONITOR] Pod $failedPodName events:")
-                println(lines.slice(eventsIndex, eventsIndex + 20).mkString("\n"))
+                logger.info(s"[MONITOR] Pod $failedPodName events:")
+                logger.info(lines.slice(eventsIndex, eventsIndex + 20).mkString("\n"))
               }
             }
           } catch {
             case e: Exception =>
-              println(s"[MONITOR] Failed to capture logs for $failedPodName: ${e.getMessage}")
+              logger.info(s"[MONITOR] Failed to capture logs for $failedPodName: ${e.getMessage}")
           }
 
           Some(s"Pod $failedPodName failed in namespace $namespace")
         } else {
           None
         }
       } else {
-        println(s"[MONITOR] Failed to get pods: ${podsResult.stderr}")
+        logger.info(s"[MONITOR] Failed to get pods: ${podsResult.stderr}")
         None
       }
     } catch {
       case e: Exception =>
-        println(s"[MONITOR] Error checking pods: ${e.getMessage}")
+        logger.info(s"[MONITOR] Error checking pods: ${e.getMessage}")
         None
     }
   }
 
   /** Capture all logs and events for debugging */
   def captureDebugInfo(): Unit = {
-    println(s"\n[DEBUG] Capturing debug info for namespace $namespace")
+    logger.info(s"[DEBUG] Capturing debug info for namespace $namespace")
```

Comment:
Instead of using the `[DEBUG]` substring here, since the logging pattern already includes the logging level, could you take out the "[DEBUG]" and just change `logger.info` to `logger.debug`?  (and same for subsequent similar ones...)